tests/utils/test_gravatar.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
```

Comment:
Do we need this extra line break?

Reply:
Those 2 line breaks are recommended by PEP8.

Reply:
Shall I remove them?